File: src/main/java/io/spring/deepdive/Application.java
package io.spring.deepdive;

import com.samskivert.mustache.Mustache;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;

import static com.samskivert.mustache.Mustache.*;

@SpringBootApplication
public class Application {

    @Bean
    public Mustache.Compiler mustacheCompiler(TemplateLoader loader) {
        return compiler().escapeHTML(false).withLoader(loader);
    }

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: src/main/kotlin/io/spring/deepdive/Application.kt
package io.spring.deepdive

import com.samskivert.mustache.Mustache.*

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.Bean
import org.springframework.data.mongodb.repository.config.EnableCoroutineMongoRepositories
import org.springframework.kotlin.experimental.coroutine.EnableCoroutine

@SpringBootApplication
@EnableCoroutine
@EnableCoroutineMongoRepositories
class Application {

    @Bean
    fun mustacheCompiler(loader: TemplateLoader) =
            compiler().escapeHTML(false).withLoader(loader)

}

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}